packages/collector/test/test_util/ProcessControls.js

```diff
@@ -143,7 +143,6 @@ class ProcessControls {
         INSTANA_LOG_LEVEL: 'warn',
         INSTANA_DISABLE_TRACING: !this.tracingEnabled,
         INSTANA_FORCE_TRANSMISSION_STARTING_AT: '1',
-        INSTANA_DEV_MIN_DELAY_BEFORE_SENDING_SPANS: opts.minimalDelay != null ? opts.minimalDelay : 0,
```

Comment:
Replaced with INSTANA_TRACING_TRANSMISSION_DELAY. There is no need to introduce a second env.